client/src/main/kotlin/no/nav/su/se/bakover/client/pdf/PdfGenerator.kt

```diff
@@ -4,13 +4,16 @@ import arrow.core.Either
 import no.nav.su.se.bakover.client.ClientError
 import no.nav.su.se.bakover.domain.SøknadInnhold
 import no.nav.su.se.bakover.domain.VedtakInnhold
+import java.util.UUID
 
 enum class Vedtakstype(val template: String) {
     AVSLAG("vedtakAvslag"),
-    INNVILGELSE("vedtakInnvilgelse")
+    INNVILGELSE("vedtakInnvilgelse"),
+    TREKK_SØKNAD("trekkSøknad")
 }
 
 interface PdfGenerator {
     fun genererPdf(søknad: SøknadInnhold): Either<ClientError, ByteArray>
     fun genererPdf(vedtak: VedtakInnhold, vedtakstype: Vedtakstype): Either<ClientError, ByteArray>
+    fun genererTrukketSøknadPdf(sakId: UUID, søknadId: UUID, vedtakstype: Vedtakstype): Either<ClientError, ByteArray>
```

Comment:
Denna behöver väl inte få in `vedtakstype`? 😊  Det finns väl för det första bara en type av "trukketSøknadPdf" och `vedtakstype` bruks i funktionen ovan fordi ett vedtak kan ha två forskjellige vedtakstyper.

Reply:
skal endres 👍